syntax="proto3";
package Transfer;
message Minimap
{
    int32 friendNumber = 1;
    message friendPositions
    {
        int32 x = 1;
        int32 y = 2;
    }
    repeated friendPositions fPositions = 2;
    int32 enemyNumber = 3;
    repeated friendPositions ePositions = 4;
}